components/ide/jetbrains/image/gha-update-image/index.ts

```diff
@@ -180,12 +208,15 @@ const updateLatestVersionsInWorkspaceaAndGradle = async () => {
 
     const majorVersion = majorVersions[0];
     console.log(`All IDEs are in the same major version: ${majorVersion}`);
+    if (skipOnDifferentBuildVersion) {
```

Comment:
Can it actually happen? Should we rather try minimal common build version and try to use it, and then integration tests should ensure that all IDEs are working?

Reply:
i.e. we go from the assumption that within the same major version there is no breaking changes

Reply:
I see 👍 thank you